kafka/producer.go

```diff
@@ -361,16 +361,42 @@ func (p *Producer) Close() {
 }
 
 const (
-	// RdKafkaPurgeFQueue - Purge from queue
-	RdKafkaPurgeFQueue = C.RD_KAFKA_PURGE_F_QUEUE
-	// RdKafkaPurgeFInflight - Purge from inflight
-	RdKafkaPurgeFInflight = C.RD_KAFKA_PURGE_F_INFLIGHT
+	// PurgeInFlight purges messages in-flight to or from the broker.
```

Comment:
This docstring should be for PurgeInFlight (note the capital F)

Provide docstrings for the other flags too (see rdkafka.h)